### DISCUSSION TITLE
Weird layout bugs in Safari

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [saadeghi/daisyui#3205] [Q&A]
My site uses TailwindCSS 3.4.10 and DaisyUI 4.12.10 and it renders great on Chrome, Firefox and Edge, but some pieces look weird on Safari 17 (macOS).

For example, this report:

https://rocketvalidator.com/s/971b8c10-be44-48c1-81a9-19e69a80072d

If you go to the bottom you'll see this:

![Captura de pantalla 2024-09-10 a las 22 19 16](https://github.com/user-attachments/assets/8f40ed54-e39d-463f-a489-eff847e43f29)

Another issue is at the footer of https://rocketvalidator.com where the theme switcher is:

![Captura de pantalla 2024-09-10 a las 22 20 05](https://github.com/user-attachments/assets/4bd440aa-f51d-4ad5-b263-1294e6fba29e)

Any idea where the problem could be?

**Comment by jaimeiniesta** [comment]
I think this is related:

https://stackoverflow.com/questions/78335449/safari-strange-issue-with-flex-and-column-and-grids

The example renders different in Safari

https://play.tailwindcss.com/uzqrEtPtKX

**Comment by saadeghi** [comment]
I don't have Safari 17 to test but I can't see any of these issues on Safari 18.1

- Are you using any extensions which may change the page content?
- Is this issue also happen in a private tab?
- Can you inspect the problematic part of the page to see what element is taking that much space or pushing other elements maybe?

**Reply by jaimeiniesta** [reply]
Hi, I've installed Safari 18.0 beta and the issue is fixed. Thanks!

For the record, in Safari 17 I didn't have any extensions and it also failed in a private tab.
